tabbycat/results/views.py

```diff
@@ -569,9 +569,13 @@ def get_queryset(self):
         ).prefetch_related('debateteam_set__team')
 
     def get(self, request, *args, **kwargs):
-        self.object = self.get_object()
+        try:
+            self.object = self.get_object()
+        except self.model.MultipleObjectsReturned:
+            error = (500, _("It looks like you were assigned to two or more debates. Please contact a tab room official."))
```

Comment:
Hmm, I'm not actually sure what the error code should be, but I feel like it should be some 4xx. Probably 404? It's not necessarily an error that the judge was assigned two or more debates (it might've been intentional), and even if it is, it's not a server error (the tab director made it like that). I guess 404 isn't necessarily semantically correct, but it's possibly the closest?

Open to discussion on this.

Reply:
A "Not Found" error is not really appropriate as the object is found, just incongruous. The 400 series is for client error, which an unsupported adj state is not. Thus it would be in the 500 series, for which 500 is the general code (and there are no more specific codes for this situation)

Reply:
Hmm, yeah, I see where you're coming from. I think I'm a little hesitant because normally I think of "500" as "omg wtf just happened", as opposed to a predictable error condition. But if you feel strongly about it I don't mind running with 500 for now; if it proves troublesome we can change it later.